packages/select/src/components/select/suggest.tsx

```diff
@@ -89,26 +89,22 @@ export class Suggest<T> extends React.PureComponent<ISuggestProps<T>, ISuggestSt
         return Suggest as new (props: ISuggestProps<T>) => Suggest<T>;
     }
 
+    public state: ISuggestState<T> = {
+        isOpen: (this.props.popoverProps && this.props.popoverProps.isOpen) || false,
```

Comment:
I prefer explicit boolean checks, because they communicate clearer intentions and lead to fewer surprises with falsey values:

```tsx
(this.props.popoverProps != null && this.props.popoverProps.isOpen) || false
```